efficientdet/third_party/grad_checkpoint/memory_saving_gradients.py

```diff
@@ -43,20 +65,20 @@ def tf_gradients(ys, *args, **kwargs):
 # specific versions we can use to do process-wide replacement of tf.gradients
 def gradients_speed(ys, xs, grad_ys=None, **kwargs):
   """Optimize for speed."""
-  return gradients(ys, xs, grad_ys, checkpoints='speed', **kwargs)
+  return gradients(ys, xs, grad_ys, checkpoints="speed", **kwargs)
```

Comment:
'?

Reply:
It is autoformating: it tries to use either " or ' for consistency.  I think this file may use " for most of cases, so it chose to use ". 
In general, our original code prefer '